dataland-frontend/src/components/resources/dataTable/cells/DataPointDisplayComponent.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <span>{{ content.displayValue.label }}</span>
+  <DocumentLink
+    label=""
+    :suffix="formattedPageNumber"
+    :download-name="content.displayValue.fileName"
+    :file-reference="content.displayValue.fileReference"
+    show-icon
+  />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import {
+  type MLDTDisplayComponentName,
+  type MLDTDisplayObject,
+} from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import DocumentLink from "@/components/resources/frameworkDataSearch/DocumentLink.vue";
+
+export default defineComponent({
+  name: "DocumentLinkDisplayComponent",
```

Comment:
I think I'm observing a display bug for displaying a Currency field with download link:
![image](https://github.com/d-fine/Dataland/assets/35459207/6c8c158e-6fad-4958-b0e4-67a85cce893b)
The Total Revenue does have a currency (JMD) but it is not displayed. Also, when formatting Currency values we should always have 2 decimal values even if the second decimal is 0.

Reply:
sure

Reply:
Fixed

Reply:
Currency value may still only have a single decimal value should be ..010.50 instead of ..010.5

![image](https://github.com/d-fine/Dataland/assets/35459207/fb56dfe0-1f6e-40c3-b195-bdbf2ceb68b5)

Reply:
If there are no decimals should it end with xyz.00 or should the decimals be dropped then?

Reply:
Rendering two decimals now in any case.